src/TreeDataGrid.tsx

```diff
@@ -292,8 +292,26 @@ function TreeDataGrid<R, SR, K extends Key>(
     // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
     const idx = column?.idx ?? -1;
     const row = rows[rowIdx];
-    if (!isGroupRow(row)) return;
```

Comment:
`row` is actually undefined on the header and summary row. I will create a follow-up PR to fix types
https://github.com/adazzle/react-data-grid/blob/main/src/DataGrid.tsx#L515

Reply:
I will also add more tests in a follow up PR